### ISSUE TITLE
SOMEONE PLEASE MAKE A VIDEO ON HOW TO INSTALL THIS SOFTWARE ON WINDOWS

### ISSUE THREAD

**Issue by jjfriend** [InstaPy/InstaPy#2238]
i didn't find any video on youtube on how to install this software for windows... please help i've been trying for almost 2 weeks. please please I beg you please someone show me a clear video that explains from a-z.

**Comment by wemakefuture** [comment]
Try to setup a vps. Do step by step guide at ubuntu its easiest. Windows is tricky - could be a issue with your path settings. You can find cheap VPS at https://www.arubacloud.com/vps/virtual-private-server-range.aspx for 1€ so you don't take any risk to waste money. If you need further help you can email me. contact@wecanfuture.com

**Comment by CharlesCCC** [comment]
Have you go through this instruction step by step ?
https://github.com/timgrossmann/InstaPy/blob/master/docs/How_to_Windows.md

**Comment by jjfriend** [comment]
thank you for your help. now when i click on quickstart.py, i'm stuck here... it says an administrator has installed google chrome on this system and it is... and when I click ok, the bottom picture appears

![untitlederr](https://user-images.githubusercontent.com/40055417/41378929-ad713b12-6f71-11e8-8e67-801988a2e40d.png)
![untitlederr](https://user-images.githubusercontent.com/40055417/41379276-a134901e-6f72-11e8-8415-97dbf53fb054.png)

**Comment by uluQulu** [comment]
Hi guys
@jjfriend those erros can be seen here and there, take it easy.
But if you fail to run the program, post the details of that problem to solve.

And yes it could be amazing to have a good tutorial for Windows, cos it is very easy to install it in Windows, I can guarantee.

Sure there are problems, too. E.g. Windows Installation DOCs will cause confusion in certain steps that needs clearance.

If that documentation could be revised, there maybe would be need for a video tutorial. But it will always be great to have.

Regards

**Comment by jjfriend** [comment]
@uluQulu it would be very very very helpful if there was a tutorial for windows.. literally alot of people use windows but the guide shown is really hard to follow and I couldn't even find one video in youtube.. not even 1!! and this is a very famous software for instagram with the most likes on github. Iv'e got an error above and I hope you know what the problem is. thank you ulu

**Comment by uluQulu** [comment]
I see, @jjfriend 
Tell a bit more about that error. e.g.,
+ Does it happen always?
+ What happens after it?
+ Does it stay at that position and do not continue?
+ Does it just show it and continue to run the program?
+ Does it make problems partially?

I have never seen that error before but I don't think it is sort of a serious problem.
If you explain the details precisely, I can help.

BTW `Chrome_WidgetWin_0` is a PID (_process ID_) of some Chrome web browser application.

**Comment by jjfriend** [comment]
ok @uluQulu this happens when I dont press ok... it shows me the errors shown in picture 1, 2, and 3 in order.  
![11](https://user-images.githubusercontent.com/40055417/41412387-e44dc8c4-6ff0-11e8-98c8-3026277ecace.png)
![2](https://user-images.githubusercontent.com/40055417/41412391-e847ffd0-6ff0-11e8-9728-b010d0212e9b.png)
![3](https://user-images.githubusercontent.com/40055417/41412396-eac9c78e-6ff0-11e8-96db-aef8d51c2f79.png)

**Comment by jjfriend** [comment]
@uluQulu here's the error when I dont press ok... it's shown in order from pictures 1, 2, 3 (from top to bottom 1-2-3).
Does it happen always? Yes
What happens after it? The pictures shown under is what happens
Does it stay at that position and do not continue? Yes, but if I press ok it closes.. If i keep it on, nothing happens and the picture below happens 
Does it just show it and continue to run the program? When I press ok, it closes
Does it make problems partially? No, always comes
![11](https://user-images.githubusercontent.com/40055417/41412977-c46bff9c-6ff2-11e8-82ac-9ec6f3612093.png)
![2](https://user-images.githubusercontent.com/40055417/41412986-ca11c800-6ff2-11e8-8f6f-06fdd7fae945.png)
![3](https://user-images.githubusercontent.com/40055417/41412988-ca72f0a8-6ff2-11e8-8166-b6815880551d.png)

**Comment by wemakefuture** [comment]
@jjfriend write me an email and i will set you up on vps contact@wecanfuture.com cant see you struggeling

**Comment by uluQulu** [comment]
It's clear @jjfriend 
Searches say it is a very fresh issue after a Chrome web browser  update.

You can try some steps
in **instapy.py** file, find 
```python
chrome_options.add_argument('--disable-setuid-sandbox')
```
line and add these guys after it one by one and test them
```python
chrome_options.add_argument('--disable-setuid-sandbox')

chrome_options.add_argument("--disable-extensions")   #disabling extensions
chrome_options.add_argument("--disable-gpu")   #applicable to windows os only
chrome_options.add_argument("--disable-dev-shm-usage")   #overcome limited resource problems
```

**Also** 
try to run the program in **headless** mode.


Of course the problem is Chrome web browser related, so downgrading it may help or getting a fresher version which has been fixed may also help.

Cheers
